src/main/java/io/openliberty/tools/intellij/lsp4mp/lsp4ij/DocumentContentSynchronizer.java

```diff
@@ -103,13 +108,33 @@ public void documentChanged(DocumentEvent event) {
         }
     }
 
+    public void documentFullRefresh(Document document) {
```

Comment:
This PR breaks the LSP concept. You should never call some LSP intenal class in an external component. If another language server have the same problem than this issue, it will require to call this method.

I need to find time to understand more the issue.